war/src/main/less/pluginSetupWizard.less

```diff
@@ -410,52 +410,126 @@
                     }
                 }
 	
+				@badge-color: #555;
```

Comment:
@kzantow two-space tab would be great, like [this](https://github.com/jenkinsci/jenkins/blob/master/war/src/main/js/widgets/add/addform.less).